explainaboard/tests/test_cli.py

```diff
@@ -148,6 +148,7 @@ def test_ner_custom(self):
         with patch('sys.argv', args):
             main()
 
+    @unittest.skip(reason='metaphor QA dataset is temporarily unavailable')
```

Comment:
We actually have metaphor QA: https://github.com/ExpressAI/DataLab/blob/main/datasets/metaphor_qa/metaphor_qa.py